# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/ipc/PSocketProcessBridge.ipdl
# Commit: d4004105a04a
# Full Hash: d4004105a04ad9a473930acb9c6a05d90cd9f028
# Author: Byron Campen [:bwc] <docfaraday@gmail.com>
# Date: 2019-02-23 04:13:22
# Regressor Bug: 1521879
# File Overlap Count: 1
# Description:
#   Bug 1521879 - Part 1: IPC-based MediaTransport implementation r=mjf
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D17273
# ==============================================================================

diff -r baecf588657c -r d4004105a04a netwerk/ipc/PSocketProcessBridge.ipdl
--- a/netwerk/ipc/PSocketProcessBridge.ipdl	Thu Feb 21 23:16:40 2019 +0000
+++ b/netwerk/ipc/PSocketProcessBridge.ipdl	Thu Feb 21 20:24:54 2019 +0000
@@ -5,6 +5,7 @@
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 include protocol PBackground;
+include protocol PMediaTransport;
 
 namespace mozilla {
 namespace net {
@@ -19,9 +20,13 @@
   */
 nested(upto inside_cpow) sync protocol PSocketProcessBridge
 {
+  manages PMediaTransport;
+
 parent:
   async InitBackground(Endpoint<PBackgroundParent> aEndpoint);
 
+  async PMediaTransport();
+
 both:
   async Test();
 };